### PR TITLE
doc: unify homepage of https://spoon.gforge.inria.fr and Github

### PR DESCRIPTION
**Pull request by monperrus**
This simplifies maintenance.

### CODE REVIEW

File: doc/faq.md [hunk 1]
```diff
@@ -5,6 +5,26 @@ keywords: frequently asked questions, FAQ, question and answer, collapsible sect
 
 ## Practical Information
 
+### Hare there snapshots versions deployed somewhere?
```
Comment (nharrand) [inline doc/faq.md]:
```suggestion
### Are there snapshots versions deployed somewhere?
```

File: README.md [hunk 2]
```diff
@@ -78,16 +78,59 @@ mvn test
...
+- December 9, 2014: Spoon 3.0 is released [(changelog)](https://lists.gforge.inria.fr/pipermail/spoon-discuss/2014-December/001721.html).
+- November 12, 2014: Spoon 2.4 is released [(changelog)](https://lists.gforge.inria.fr/pipermail/spoon-discuss/2014-November/001710.html).
+- October 9, 2014: Spoon 2.3.1 is released [(changelog)](https://lists.gforge.inria.fr/pipermail/spoon-discuss/2014-October/001688.html).
+- September 12, 2014: Spoon 2.1 is released [(changelog)](https://lists.gforge.inria.fr/pipermail/spoon-discuss/2014-September/001683.html).
+- April 2, 2014: Spoon 2.0 is released [(changelog)](https://lists.gforge.inria.fr/pipermail/spoon-discuss/2014-March/001639.html).
+- September 30, 2013: Spoon 1.6 is released.
+- April 12, 2012: Spoon 1.5 is released.
 
-  - [Spoon: Getting Started - Simon Urli @ OW2Con'18 (Paris)](https://www.youtube.com/watch?v=ZZzdVTIu-OY)
-  - [Generate Test Assertion with Spoon - Benjamin Danglot @ OW2Con'17 (Paris)](https://www.youtube.com/watch?v=JcCIbjnkfD4)
```
Comment (nharrand) [inline README.md]: Can't we keep these links toward videos? (I mean we could add them on the home page rather than deleting them from the readme.)

Comment (monperrus) [reply README.md]: good idea, done.

**Comment by nharrand** [comment]
LGTM.

**Pull request merged**